docs/13-walkthroughs/6-publish-and-deploy.md

```diff
@@ -145,75 +70,99 @@ To modify, edit `zarf.yaml` and re-run `zarf package create .`
 
 Inspecting a Zarf package stored in an OCI registry is the same as inspecting a local package and has the same flags:
 
-```yaml
-$ zarf package inspect oci://$REPOSITORY_URL/helm-oci-chart:0.0.1-arm64
----
-kind: ZarfPackageConfig
-metadata:
-  name: helm-oci-chart
-  description: Deploy podinfo using a Helm OCI chart
-  version: 0.0.1
-  architecture: arm64
-build:
-  terminal: minimind.local
-  user: whoami
-  architecture: arm64
-  timestamp: Tue, 07 Mar 2023 14:27:25 -0600
-  version: v0.25.0-rc1-41-g07d61ba7
-  migrations:
-    - scripts-to-actions
-components:
-  - name: helm-oci-chart
-    required: true
-    charts:
-      - name: podinfo
-        url: oci://ghcr.io/stefanprodan/charts/podinfo
-        version: 6.3.3
-        namespace: helm-oci-demo
-    images:
-      - ghcr.io/stefanprodan/podinfo:6.3.3
-```
-
-## Deploy Package
+<iframe src="/docs/walkthroughs/publish_and_deploy_inspect.html" width="100%" height="550px"></iframe>
 
-[CLI Reference](../4-user-guide/1-the-zarf-cli/100-cli-commands/zarf_package_deploy.md)
 
-Deploying a package stored in an OCI registry is nearly the same experience as deploying a local package:
+### Deploy Package
 
-```bash
-# Due to the length of the console output from this command,
-# it has been omitted from this walkthrough
-$ zarf package deploy oci://$REPOSITORY_URL/helm-oci-chart:0.0.1-arm64
-# Make these choices at the prompts:
-# Deploy this Zarf package? Yes
+[CLI Reference](../4-user-guide/1-the-zarf-cli/100-cli-commands/zarf_package_deploy.md)
 
-$ zarf packages list
+Deploying a package stored in an OCI registry is nearly the same experience as deploying a local package:
 
-    Package        | Components
-    helm-oci-chart | [helm-oci-chart]
-    init           | [zarf-injector zarf-seed-registry zarf-registry zarf-agent git-server]
-```
+<iframe src="/docs/walkthroughs/publish_and_deploy_deploy.html" width="100%" height="800px"></iframe>
 
-## Pull Package
+### Pull Package
 
 [CLI Reference](../4-user-guide/1-the-zarf-cli/100-cli-commands/zarf_package_pull.md)
 
 Packages can be saved to the local disk in order to deploy a package multiple times without needing to fetch it every time.
 
-```bash
-# go home so we don't clobber our currently local built package
-$ cd ~
-$ mkdir -p zarf-packages && cd zarf-packages
+<iframe src="/docs/walkthroughs/publish_and_deploy_pull.html" width="100%" height="450px"></iframe>
+
+
+## Removal
+
+1. Use the `zarf package list` command to get a list of the installed packages.  This will give you the name of the games package to remove it.
+
+<iframe src="/docs/walkthroughs/publish_and_deploy_list.html" width="100%"></iframe>
+
+2. Use the `zarf package remove` command to remove the `helm-oci-chart` package.  Don't forget the `--confirm` flag.  Otherwise you'll receive an error.
+
+<iframe src="/docs/walkthroughs/publish_and_deploy_remove.html" width="100%" height="115px"></iframe>
+
```

Comment:
Doesn't need to be addressed now, but we will likely want to discuss this not being repeated in every walkthrough (since most will just be `zarf package list` `zarf package remove`